src/pages/Frontpage.jsx

```diff
@@ -127,9 +122,19 @@ function Frontpage(props) {
         />
         <br></br>
         <br></br>
+
+        {/* Dodaj istniejący przycisk "Submit" */}
         <button className="main primary" href={`${props.id}`}>
           Submit
         </button>
+
+        {/* Nowy input dla ownera */}
+        <label>Owner</label>
+        <input
+          {...register("owner", { required: true })}
+          placeholder="Enter owner name"
+        />
+        {errors.owner && <div className="text-red-500 text-sm">Owner is required</div>}
```

Comment:
Input na ownera można dać nad submitem tak żeby submit był na samym dole (w większości formularzy tak się dzieje)

btw. labelkę, placeholder i errora można zmienić na język polski jak w reszcie apki

zamiast Owner czy tam "Właściciel" można to pole nazwać po prostu "Imię" albo "Twoje imię"